### PR TITLE
handle unicode in intervals and start logging when "auto" intervals are

### PR DESCRIPTION
**Pull request by elainearbaugh**
used (to see if we can start disallowing auto intervals)

### CODE REVIEW

File: cabot/metricsapp/api/grafana_elastic.py [hunk 1]
```diff
@@ -88,8 +88,9 @@ def _get_date_histogram_settings(agg, min_time, default_interval):
     :param default_interval: default for the group by interval
     :return: dict of {setting_name: setting_value}
     """
-    interval = agg['settings']['interval']
+    interval = str(agg['settings']['interval'])
```
Comment (EvanKuhn-Affirm) [inline cabot/metricsapp/api/grafana_elastic.py]: I notice that calling `str()` on a unicode string that can't be converted to ascii will raise an error:

```
>>> s = u'xxàuu'
>>> type(s)
<type 'unicode'>
>>> str(s)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe0' in position 2: ordinal not in range(128)
```

Perhaps use `.encode('ascii', 'ignore')` ?

```
>>> s.encode('ascii', 'ignore')
'xxuu'
```

Or is this not really an issue in our case?

https://docs.python.org/2/howto/unicode.html#the-unicode-type

Comment (elainearbaugh) [reply cabot/metricsapp/api/grafana_elastic.py]: This is for the date histogram group by interval (1m, 2m, 1h, etc) in Grafana, which shouldn't ever have special characters (and if it did the dashboard would be erroring out anyways and the ES query we create would also error out) so I think it's ok to not handle special cases in this circumstance

Comment (elainearbaugh) [reply cabot/metricsapp/api/grafana_elastic.py]: def a good thing to keep in mind in general though, thanks for pointing it out

Comment (EvanKuhn-Affirm) [reply cabot/metricsapp/api/grafana_elastic.py]: ok, thanks!

**Comment by elainearbaugh** [comment]
trigger build

**Pull request merged**